tests/utils/test_helpers.py

```diff
@@ -70,3 +71,10 @@ def test_get_client_cert(config):
     config.merge({"certificates": {"foo": {"client-cert": client_cert}}})
 
     assert get_client_cert(config, "foo") == Path(client_cert)
+
+
+def test_get_trusted(config):
+    trusted = "true"
+    config.merge({"certificates": {"foo": {"trusted": trusted}}})
+
+    assert get_truested(config, "foo")
```

Comment:
there seems to be a typo here as well: get_trusted

Reply:
Fixed :)

Reply:
Thanks! However it seems more must be done as part of this change:
` RuntimeError: The Poetry configuration is invalid:
E             - [source.0] Additional properties are not allowed ('trusted' was unexpected)
`

Reply:
This is expected, as written in the original post, in order for tests to pass you must first complete the mentioned PR in `poetry-core` as it contains the required changes to the poetry configuration schema.